internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml

```diff
@@ -447,4 +447,14 @@ Resources:
                 Action: acm:DescribeCertificate
                 Resource: "*"
       ManagedPolicyArns:
-        - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
\ No newline at end of file
+        - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+
+Outputs:
+  CloudFrontDistributionDomainName:
+    Value: !GetAtt CloudFrontDistribution.DomainName
+    Export:
+      Name: !Sub ${AWS::StackName}-CloudFrontDistributionDomainName
+  CloudFrontDistributionAlternativeDomainName:
+    Value: "example.com"
+    Export:
+      Name: !Sub ${AWS::StackName}-CloudFrontDistributionAlternativeDomainName
```

Comment:
do you know if there is a limit in the `Export.Name` length?

Reply:
255 characters. We can adjust accordingly to truncate "-CloudFrontDistributionAlternativeDomainName" as needed in the future.